phpcs-ruleset.xml.dist

```diff
@@ -30,4 +32,51 @@
     <rule ref="Drupal.Files.TxtFileLineLength.TooLong">
         <exclude-pattern>web/modules/custom/joinup_subscription/README.md</exclude-pattern>
     </rule>
+
+    <!-- Require the strict types declaration in every PHP file. -->
+    <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
+        <properties>
+            <property name="newlinesCountBetweenOpenTagAndDeclare" value="2"/>
+        </properties>
+    </rule>
+
+    <!-- Require the strict types declaration in every PHP file. -->
+    <rule ref="SlevomatCodingStandard.TypeHints.NullableTypeForNullDefaultValue">
+    </rule>
```

Comment:
The comment on this has been copied from the previous sniff, it needs to be updated.